backend/src/schema/index.js

```diff
@@ -35,14 +36,14 @@ export default applyScalars(
             'Notfication',
             'Post',
             'Comment',
-            'Report',
             'Statistics',
             'LoggedInUser',
             'Location',
             'SocialMedia',
             'User',
             'EMOTED',
             'NOTIFIED',
+            'REPORTED',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T09:32:57Z" title="Monday, October 14th 2019, 11:32:57 am +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,14 +35,14 @@ export default applyScalars(
             'Notfication',
             'Post',
             'Comment',
-            'Report',
             'Statistics',
             'LoggedInUser',
             'Location',
             'SocialMedia',
             'User',
             'EMOTED',
             'NOTIFIED',
+            'REPORTED',
```


I guess that you want to exclude the `REPORTED` query as well, otherwise we have one for `reports` and another for `REPORTED`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T13:24:01Z" title="Monday, October 14th 2019, 3:24:01 pm +02:00">Oct 14, 2019</time>_

---


Yes, genau!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T13:59:39Z" title="Monday, October 14th 2019, 3:59:39 pm +02:00">Oct 14, 2019</time>_

---


sorry, you marked as resolved, but didn't exclude `REPORTED` for queries... maybe by accident?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T15:16:47Z" title="Monday, October 14th 2019, 5:16:47 pm +02:00">Oct 14, 2019</time>_

---


I did locally not yet pushed up.
You mean to exclude it from the queries list, I suppose?